code/noRAM/app/src/main/res/layout/attendee_event_info.xml

```diff
@@ -17,7 +17,7 @@
         app:layout_constraintTop_toTopOf="parent" />
 

```

Comment:
also, as a general note, the "announcements" text by the bell should be vertically centered with the bell, and isnt. I also see the bell in two places?

Reply:
I didnt do anything on this file

Reply:
I will remove the other bell tho thats my bad